lib/log_parser.rb

```diff
@@ -18,7 +19,8 @@ def first_line
   end
 
   def generate_json
-    hash_json = { "#{@log_file}": { "lines": count_lines } }
+    id_players
+    hash_json = { "#{@log_file}": { "lines": count_lines, "players": @players } }
```

Comment:
nome de variavél muito genérico, prefira algo que descreva o que está sendo armazenado na variável do que o tipo dela
```suggestion
    game_info = { "#{@log_file}": { "lines": count_lines, "players": @players } }
```